ethers-contract/ethers-contract-abigen/src/source/online.rs

```diff
@@ -182,9 +179,6 @@ mod tests {
 
         let explorers = &[
             ("mainnet:", "etherscan:", "https://etherscan.io/address/", Network::Mainnet),
-            ("bsc:", "bscscan:", "https://bscscan.com/address/", Network::BinanceSmartChain),
-            ("polygon:", "polygonscan:", "https://polygonscan.com/address/", Network::Polygon),
-            ("avalanche:", "snowtrace:", "https://snowtrace.io/address/", Network::Avalanche),
```

Comment:
There should be Devin added probably. Do we have a block explorer for the Devin network?